src/handle_action.cpp

```diff
@@ -1273,7 +1273,8 @@ static void takeoff()
     item_location loc = game_menus::inv::take_off( player_character );
 
     if( loc ) {
-        player_character.takeoff( *loc.obtain( player_character ) );
+        loc.obtain( player_character );
```

Comment:
`obtain` may move the item so I think the returned `item_location` should be passed to `takeoff` here?

Reply:
Thanks, I am getting confused on which changes to item_location are made in place and which you have to catch in the return.